README.md

```diff
@@ -21,12 +20,13 @@ module "security_hub_collector_runner" {
   ecs_vpc_id           = data.aws_vpc.mac_fc_example_east_sandbox.id
   ecs_subnet_ids       = [data.aws_subnet.private_a.id]
   ecs_cpu              = // optional, defaults to 256
-  ecs_memory           = // optionals, defaults to 1024
-  assign_public_ip     = true // optional, defaults to false
+  ecs_memory           = // optional, defaults to 1024
+  assign_public_ip     = // optional, defaults to false
```

Comment:
Do we know why this was originally set to true?

Reply:
No, I don't know.  Since the default is false, then it makes sense to me for the example to show that without an override.  That way if someone copy-pastes it they will get the default unless they intentionally override it.

Reply:
Based on what this task does, I can't see how it is required, so this change should be okay.